### PR TITLE
test: enable sanity test for vhd disk feature

### LINKED ISSUE #182: enable sanity test for vhd disk feature
**Is your feature request related to a problem?/Why is this needed**

**Describe the solution you'd like in detail**

Current sanity test is mainly for azure file share driver, not for vhd disk feature, need to enable those sanity tests for vhd disk feature
https://github.com/kubernetes-sigs/azurefile-csi-driver/tree/master/test/sanity

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by ZeroMagic**
**What type of PR is this?**
/kind test

**What this PR does / why we need it**:
Add a parameter yaml file to enable sanity test for vhd disk feature.

**Which issue(s) this PR fixes**:

Fixes #182 

**Special notes for your reviewer**:

**Release note**:
```
enable sanity test for vhd disk feature
```

### CODE REVIEW

File: test/sanity/run-test.sh [hunk 1]
```diff
@@ -30,13 +30,21 @@ nodeid='CSINode'
...
 else
-  "$CSI_SANITY_BIN" --ginkgo.v --ginkgo.noColor --csi.endpoint="$endpoint"  --ginkgo.skip='should fail when the volume source snapshot is not found|should work'
+  "$CSI_SANITY_BIN" --ginkgo.v --ginkgo.noColor --csi.endpoint="$endpoint" --ginkgo.skip='should fail when the volume source snapshot is not found|should work'
+fi
+
+echo 'Begin to run sanity test for vhd disk feature...'
+if [[ "$TRAVIS" == 'true' ]]; then
+  "$CSI_SANITY_BIN" --ginkgo.v --ginkgo.noColor --csi.endpoint="$endpoint" --csi.testvolumeparameters="$testvolumeparameters" --ginkgo.skip='should work'
+else
+  "$CSI_SANITY_BIN" --ginkgo.v --ginkgo.noColor --csi.endpoint="$endpoint" --csi.testvolumeparameters="$testvolumeparameters" --ginkgo.skip='should fail when the volume source snapshot is not found|should work'
```
Comment (andyzhangx) [inline test/sanity/run-test.sh]: does it support non-file parameter?

Comment (ZeroMagic) [reply test/sanity/run-test.sh]: No, it just supports the parameter file.

Comment (andyzhangx) [reply test/sanity/run-test.sh]: I think you could use shell to create a dyanmic file in sanity testing

Comment (ZeroMagic) [reply test/sanity/run-test.sh]: Ref: https://github.com/kubernetes-csi/csi-test/blob/master/cmd/csi-sanity/main.go#L85

**Comment by ZeroMagic** [comment]
It seems that the actual deleting operation costs more time to complete. The related doc also reminds the user. 
https://docs.microsoft.com/zh-cn/rest/api/storageservices/create-share#remarks

In the doc, it is status code 409 (Conflict). But here it is status code 404 (Not Found).

**Pull request merged**